.vscode/settings.json

```diff
@@ -0,0 +1,65 @@
+{
```

Comment:
.vscode keep finding it's way back in here. You need to be aware of what files are being committed. The .vscode should be added to the .gitigore and this file needs to be removed/stopped being tracked